setup.py

```diff
@@ -33,20 +30,21 @@
     maintainer_email='dev-platform@dropbox.com',
     url='https://github.com/dropbox/stone',
     install_requires=install_reqs,
-    license=LICENSE,
+    license=open('LICENSE'),
```

Comment:
Apparently both `license` and `long_description` [take file objects as arguments](http://setuptools.readthedocs.io/en/latest/setuptools.html#metadata). Who knew?